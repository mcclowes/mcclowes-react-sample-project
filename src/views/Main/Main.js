import Dumb from "../../components/Dumb";
import React from "react";
import Page from "../../components/Page";
import Smart from "../../components/Smart";
import { Text } from "../../components/UIKit";

const Main = (props) => {
  return (
    <Page>
      <Smart>
        <Text>Hello World!</Text>
      </Smart>

      <Dumb>
        <Text>This is an example project. Make changes as you see fit.</Text>
      </Dumb>
    </Page>
  );
};

export default Main;
