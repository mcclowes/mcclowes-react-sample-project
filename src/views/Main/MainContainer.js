import React from "react";
import Main from "./Main";

const MainContainer = (props) => {
  return <Main {...props} />;
};

export default MainContainer;
