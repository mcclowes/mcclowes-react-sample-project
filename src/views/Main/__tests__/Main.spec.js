import Main from "../Main";
import React from "react";
import { shallow } from "enzyme";

const defaultProps = {};

const renderComponent = (newProps) => {
  const props = {
    ...defaultProps,
    ...newProps,
  };

  return shallow(<Main {...props} />);
};

let wrapper;

it("renders component", () => {
  wrapper = renderComponent();

  expect(wrapper).toMatchSnapshot();
});
