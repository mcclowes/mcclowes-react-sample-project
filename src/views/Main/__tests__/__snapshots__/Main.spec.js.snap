// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<Page>
  <SmartContainer>
    <Text>
      Hello World!
    </Text>
  </SmartContainer>
  <Dumb>
    <Text>
      This is an example project. Make changes as you see fit.
    </Text>
  </Dumb>
</Page>
`;
